src/Microsoft.DotNet.ScenarioTests.SdkTemplateTests/SdkTemplateTests.cs

```diff
@@ -37,6 +37,12 @@ public void VerifyConsoleTemplateComplex(DotNetLanguage language)
     [Trait("SkipIfBuild", "CommunityArchitecture")] // Portable assets are not available for community architectures.
     public void VerifyConsoleTemplateComplexPortable(DotNetLanguage language)
     {
+        // Skip if a portable RID isn't provided.
+        if (_scenarioTestInput.PortableRid is null)
```

Comment:
Rather than silent skipping, perhaps we should make `PortableRid` non-nullable and either throw an exception or return some non-sense string like `error-portable-rid-not-set`?

Reply:
We do have logic to calculate the portable RID in the  VMR: https://github.com/dotnet/dotnet/blob/a372a618aaf8f54306234a625459cf86395c508f/Directory.Build.props#L67-L72

I didn't want to bring that over as this repo doesn't use some of the dependent properties. We have a tracking issue to move the definition of these Build,Host,Target (including the portableRid) properties into the Arcade SDK so that repos don't need to calculate those themselves.

FWIW I could bring the existing default value (which is totally wrong) back. Given that the VMR will set this value correctly, that might not be an issue for now and would definitely not be worse than the current state in main.

Reply:
> FWIW I could bring the existing default value (which is totally wrong) back. Given that the VMR will set this value correctly, that might not be an issue for now and would definitely not be worse than the current state in main.

Sounds good.

fyi, to determine the portable rid for linux-musl and linux-bionic you need some info from the native build script: https://github.com/search?q=repo%3Adotnet%2Fruntime%20__PortableTargetOS&type=code.